docs/tools/offline/cli-test-debug.md

```diff
@@ -91,6 +124,17 @@ mbedgt: available tests for built 'K64F-ARM', location '.\build\tests\K64F\ARM'
         test 'TESTS-functional-test3'
 ```
 
+In case of icetea:
```

Comment:
Same ? as above: what do you mean by "in case of icetea"? Do you mean to 'to find the tests that are available' for it, as in the previous example?